packages/ui/src/icons/all-icons.web.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { Meta } from '@storybook/react';
+
+import * as Icons from './index.web';
```

Comment:
Do you think we could do this in the `extension` instead? 

That storybook gets published so the designers can see it. Since moving icons to the monorepo they don't have a way to see them without building the UI package.

We discussed composing storybooks between `mono` + `extension` but never did it

Reply:
Maybe we can try to export `all-icons.web.stories.tsx` from `mono`? 

Then we could just load the story into the `extension` storybook? 🤔  

Not something for now but maybe for the future